src/prefect/server/schemas/responses.py

```diff
@@ -261,13 +265,33 @@ class DeploymentResponse(ORMBaseModel):
         default=None,
         description="The name of the deployment's work pool.",
     )
+    status: Optional[schemas.statuses.DeploymentStatus] = Field(
+        default=None,
+        description="Current status of the deployment.",
+    )
     enforce_parameter_schema: bool = FieldFrom(schemas.core.Deployment)
 
     @classmethod
     def from_orm(
         cls, orm_deployment: "prefect.server.database.orm_models.ORMDeployment"
     ):
         response = super().from_orm(orm_deployment)
+
+        offline_horizon = datetime.datetime.now(
+            tz=datetime.timezone.utc
+        ) - datetime.timedelta(seconds=DEPLOYMENT_LAST_POLLED_TIMEOUT_SECONDS)
```

Comment:
happy to rename this to something like `not_ready_horizon` if folks don't think this is aptly named

Reply:
Yeah, I prefer `not_ready_horizon` since it uses the established nomenclature for deployment status.

Reply:
Renamed in [a0b989b](https://github.com/PrefectHQ/prefect/pull/10951/commits/a0b989beb477762efdf76906b82296b75dfca8d6)